ci/docker/install/ubuntu_install_mxnet.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -ex
+
+YEAR=$(date +"%Y")
+MONTH=$(date +"%m")
+DATE=$(date +"%d")
+
+pip3 install https://apache-mxnet.s3-us-west-2.amazonaws.com/dist/"${YEAR}"-"${MONTH}"-"${DATE}"/dist/mxnet_cu101mkl-1.6.0b"${YEAR}""${MONTH}""${DATE}"-py2.py3-none-manylinux1_x86_64.whl
```

Comment:
I'm vetoing the usage of that s3 bucket, see the email thread for further reasons.

Reply:
```
 The solution for the initial issue - no longer publishing
to pypi - is to add a step to the existing CD pipeline which publishes the
package to the s3 bucket instead of pypi.
```

Hi @marcoabreu Reading your email thread, I understand that you have concerns about replacing Jenkins CD with Codebuild, but you are fine with publishing mxnet nightly to S3 bucket. This PR is not related to changing current CD pipeline, but using a mxnet nightly wheel to run nightly tests, which is missing today. If I misunderstood your concern, please let me know. Thanks for your passion about this project.

Reply:
Hey @marcoabreu agree on @apeforest The PR is not about where we upload our nightly binaries, but test what we have been publishing. If we decide to go for PyPi in the future, I would love to raise another PR to fix that.

Reply:
Correct, the issue is not the s3 bucket. Excuse me, but I'm "hijacking" this PR (and other PRs related to the s3 bucket) to enforce the veto. The goal is to avoid this unapproved solution from being further integrated until it's conform again. Once the publish happens through the MXNet CD, I'm happy to remove my veto. Please excuse the caused inconveniences.